test_tracetools/test/test_client.py

```diff
@@ -39,11 +42,25 @@ def test_all(self):
         self.assertEventsSet(self._events_ros)
 
         # Check fields
+        rmw_client_init_events = self.get_events_with_name(tp.rmw_client_init)
         client_init_events = self.get_events_with_name(tp.rcl_client_init)
```

Comment:
just a minor suggestion. how about adding `rcl` prefix here too? at least for me it would be easier to understand the order, and other places?

```suggestion
        rcl_client_init_events = self.get_events_with_name(tp.rcl_client_init)
```

Reply:
that's totally fair; fixed!